server/cmd/server.go

```diff
@@ -53,7 +53,6 @@ func serveRun(cmd *cobra.Command, args []string) {
 	// Set routing rules
```

Comment:
This is old server.go code. Please remove this file from your pr :)
use
`git checkout main -- server/cmd/server.go`